docs/sharding.md

```diff
@@ -0,0 +1,107 @@
+# Multiple Azure SignalR Service instances Support
+Currently we add support for configuring multiple SignalR Service instances under **persistent** mode. You can distribute your clients to multiple SignalR service instances and send messages to multiple instances as if to one instance. 
+
+<!-- TOC -->
+
+- [Multiple Azure SignalR Service instances Support](#multiple-azure-signalr-service-instances-support)
+  - [Usage Scenarios](#usage-scenarios)
+  - [Configuration method](#configuration-method)
+  - [Route algorithm](#route-algorithm)
+    - [Default behavior](#default-behavior)
+    - [Customization](#customization)
+    - [Implicitly pass HttpContext to endpoint router](#implicitly-pass-httpcontext-to-endpoint-router)
+
+<!-- /TOC -->
+
+## Usage Scenarios
+Routing logic is the way to decide to which SignalR Service instance among multiple instances your clients connect and your messages send. By applying different routing logic, this feature can be used in different scenarios. 
+* Scaling. Randomly route each client to one SignalR Service instance, send messages to all the SignalR Service instances so that you can scale the concurrent connections.
```

Comment:
add link to the details section in azure docs

Reply:
Related sections in Azure docs are about Server mode. Would it be confusing for users?

Reply:
add a notice to mention the difference, and similarity about app server and function app